ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1385,13 +1539,147 @@
           status_context: jenkins-kind-ipv6-ds-e2e
           status_url: --none--
           success_status: Build finished.
-          failure_status: Failed. Add comment /test-kind-aws-ipv6-e2e to re-trigger.
-          error_status: Failed. Add comment /test-kind-aws-ipv6-e2e to re-trigger.
+          failure_status: Failed. Add comment /test-kind-ipv6-e2e to re-trigger.
+          error_status: Failed. Add comment /test-kind-ipv6-e2e to re-trigger.
           triggered_status: null
           started_status: null
           wrappers:
           - timeout:
               fail: true
               timeout: 135
               type: absolute
-          publishers: []
+          publishers:
+          - archive:
+              allow-empty: true
+              artifacts: antrea-test-logs.tar.gz
+              case-sensitive: true
+              default-excludes: true
+              fingerprint: false
+              only-if-success: false
+      - '{name}-{test_name}-for-pr':
+          test_name: kind-all-features-conformance
+          node: 'antrea-kind-testbed'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+          - ${{sha1}}
+          builders:
+            - builder-conformance-kind:
+               conformance_type: 'conformance'
+               ip_version: 'ipv4'
+               kind_cluster_name: '{test_name}'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-(all-features-conformance).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-kind-all-features-conformance
+          status_url: null
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-kind-all-features-conformance to re-trigger.
+          error_status: Failed. Add comment /test-kind-all-features-conformance to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - credentials-binding:
+            - text:
+                credential-id: CODECOV_TOKEN
+                variable: CODECOV_TOKEN
+          - timeout:
+              fail: true
+              timeout: 80
+              type: absolute
+          - credentials-binding:
+            - text:
+                credential-id: CAPVC_USERNAME
```

Comment:
These credentials are only needed for CAPV jobs, we can remove them in all kind jobs.